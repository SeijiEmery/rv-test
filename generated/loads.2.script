load /x 0 generated/loads.2.hex
writereg 10 -7
setpc 0
run 1004
readreg 11
readreg 12
readreg 13
readreg 14
readreg 15
readreg 16
readreg 17