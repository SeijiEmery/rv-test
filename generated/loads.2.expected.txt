tests/loads.test.s:39
R11 = -7
R12 = 4294967289
R13 = 65529
R14 = 249
R15 = 18446744073709551609
R16 = 18446744073709551609
R17 = 18446744073709551609
